utils.js

```diff
@@ -42,5 +42,7 @@ function messageResponse (name, message) {
 
 module.exports = {
   parseMessage,
-  messageResponse
+  messageResponse,
+  printGeneralHelp,
```

Comment:
I personally feel using messageResponse would be too high level. If you say, I would change the tests to use messageResponse. I will submit a commit which these changes later today.